### ISSUE TITLE
DiskUnrecognizedException when using acx on .dsk file

### ISSUE THREAD

**Issue by marktmiller** [AppleCommander/AppleCommander#90]
I've been trying to use acx to extract a file from a .dsk image, and I'm getting this exception. According to the documentation I'm reading on this project, it should be able to open a .dsk file. I'm able to open it on AppleWin and MicroM8 emulators. I've tried seeing if it's a matter of permissions (whether acx is not being allowed to access a directory, or access the .dsk file), and it doesn't appear to be that.

The command line I'm using is: java -jar AppleCommander-acx-1.8.0.jar x -d=utidsk1.dsk week.app

"week.app" is what I'm trying to extract.

I'm running acx on macOS 13.1 on Intel, and Java 11.

**Comment by ryandesign** [comment]
Yes, that should work. Are you able to see a directory listing of this disk by using:

```sh
java -jar AppleCommander-acx-1.8.0.jar ls -d=utidsk1.dsk
```

If so, does "week.app" appear in the directory listing?

I suspect this is not a type of disk image file that AppleCommander understands. Yes, it reads "dsk" files, but "dsk" is a fairly generic extension shared by many different systems, and of course anybody can rename a file to have a "dsk" extension, which does not magically make it an AppleCommander-compatible "dsk" file.

What kind of disk image do you believe this to be? AppleCommander supports DOS, ProDOS, Pascal and other types of disk images with various different track ordering schemes, but maybe you have yet another type of disk image that it doesn't know about. For example, "woz" is a newer Apple II disk image format that I don't see any mention of in AppleCommander's code.

What do you get if you run the `file` command on it?

```sh
file utidsk1.dsk
```

**Comment by a2geek** [comment]
I tried on mine and it appears to be case sensitive. So maybe type the name exactly as it appears...
